File: Models/UserRole.cs
﻿#nullable disable

namespace CRMService.Models
{
    public class UserRole
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public List<User> RoleOwners { get; set; }
        public int RuleId { get; set; }
        public List<ActionAccess> Rules { get; set; }

    }
}

File: ClientApp/src/app/api/models/user-role.ts
/* tslint:disable */
/* eslint-disable */
import { ActionAccess } from '../models/action-access';
import { User } from '../models/user';
export interface UserRole {
  id?: number;
  name?: string | null;
  roleOwners?: Array<User> | null;
  ruleId?: number;
  rules?: Array<ActionAccess> | null;
}
